ddtelemetry/src/worker.rs

```diff
@@ -441,6 +450,18 @@ impl TelemetryWorkerHandle {
             .register_metric_context(name, tags, metric_type, common, namespace)
     }
 
+    pub async fn send_msg(&self, msg: TelemetryActions) -> Result<()> {
+        Ok(self.sender.send(msg).await?)
```

Comment:
Do we want to use send here, or try_send? 
`send` will block until the receiver has space for the message, which is probably not what we want